chart/src/App.vue

```diff
@@ -106,6 +169,22 @@ export default {
       return(this.selected_län == 'Alla' ? null : this.selected_län);
     },
 
+    graphType () {
+      if (this.selected_valueType === 'total') {
+        if (this.selected_emissionType === 'all') {
+          return 0;
+        } else {
+          return 1;
+        }
```

Comment:
```suggestion
        this.selected_emissionType === 'all' ? 0 : 1
```

A bit better, maybe, but mapping to indexes can be hard to follow.  It might be better if `graphType` simply returned something like:

```javascript
`${this.selected_valueType}_${this.selected_emissionType}`
```

Then there could be an object with keys that correspond to those outputs which can be drawn from.

Also, maybe we'll disagree on this, but I try to avoid `camelCase` whenever possible.  Sometimes I work in a project with a number of other people that use it and I just have to go with it, but since I started working on this myself I went with snake case (things from outside the app are camel case because that's the javascript convention).  We could talk about if we should establish one or the other, but mixing (like `selected_valueType`) should probably be avoided

Reply:
snake_case: we can talk about why you should use one over another, but I agree mixing them is definitely not a good idea. I realized it was wrong as I was doing it, but I'm so used to camelCase in JavaScript that I couldn't help myself 😄 I've fixed this in a commit I just pushed.
As this project is already using snake_case, I think it would be best to stick with it.

I agree that this function isn't great. I've updated it to use the approach you've suggested. My concern with that approach before was that the mapping may be a bit magical, but I think that's not true anymore if it's kept inside of this function.